src/App.css

```diff
@@ -957,36 +959,32 @@ body,
   }
 }
 
-@media (max-width: 1000px) {
+
+/* iPad Mini - Landscape */
+/* Samsung Galaxy Tab S8 - Landscape */
+@media (max-width: 1040px) {
   .earth_orbits_container {
-    left: 27vw;
+    top: -10vh;
   }
-}
 
-@media (max-width: 750px) {
-  .splash_screen_body {
-    width: 90%;
-  }
```

Comment:
This breakpoint was pushed down to the bottom by other breakpoints so this looks like  a mess.